pyramid_multiauth/__init__.py

```diff
@@ -37,10 +37,11 @@ def track_policy(event):
             print("We selected policy %s" % event.policy)
 
     """
-    def __init__(self, policy, request):
+    def __init__(self, policy, request, userid):
```

Comment:
Changing this changes the API in an incompatible way. We might want to have this default to `None`.